src/clojure/nrepl/core.clj

```diff
@@ -228,6 +228,23 @@
   (throw (IllegalArgumentException.
           (format "No nREPL support known for scheme %s, url %s" (uri-scheme uri) uri))))
 
+(defmulti url-scheme
+  "Show which scheme is associated with a transport var"
+  identity)
+
+(defmethod url-scheme #'transport/bencode
```

Comment:
Those two `defmethods` can potentially go where we also define `url-connect` for the built-in transports.

Reply:
You mean next to each other? Did it now o/